### ISSUE TITLE
.bat files throw IllegalArgumentException (.txt files in .zip are in unix style)

### ISSUE THREAD

**Issue by jo-pol** [jo-pol/DiBL#7]
# the problem
following the procedure on http://jo-pol.github.io/DiBL/grounds/ with a .sh file works fine but a .bat file results in:
```
Exception in thread "main" java.lang.IllegalArgumentException: 3
expecting two numbers on the first line with the dimensions of the matrix
        at dibl.Main.readMatrix(Main.java:122)
        at dibl.Main.<init>(Main.java:67)
        at dibl.Main.main(Main.java:54)
```
# analysis
https://github.com/jo-pol/DiBL/releases/download/tiles-0.1.3/dibl-tiles-0.1.3-bin.zip
contains .txt files in the input folder that are in unix format. The program does not recognize the lines when launched with a .bat file.

# solution
fix either the two occurrences of reader.readLine() in standalone/tiles/dibl-tiles/src/main/java/dibl/math/Matrix.java or package the files in windows style.

**Comment by jo-pol** [comment]
fixed with [0.1.3b.zp](https://github.com/jo-pol/DiBL/releases/download/tiles-0.1.3/dibl-tiles-0.1.3b.zip)

**Comment by jo-pol** [comment]
This issue was moved to jo-pol/dibl-tensioned#7
